backend/Match/views.py

```diff
@@ -15,7 +15,7 @@ class MatchView(APIView):
     @method_decorator(must_be_authenticated)
     def get(self, request):
         token_str = request.query_params.get('token')
-        if not token_str:
+        if not token_str or token_str == 'null':
```

Comment:
Does python treat the null as true?

Reply:
nope, this is a hack, the frontend given it does not find the token will send a string 'null' to the back

Reply:
ohhhh